README.md

```diff
@@ -196,21 +222,17 @@ A collection of awesome things that use or support [MapLibre](https://maplibre.o
 
 **In bold**: Members of the [MapLibre Sponsorship Program](https://maplibre.org/sponsors/)
 
-## Utilities
+## Map Tile Generators
```

Comment:
Tile Serving? Tile Servers?
There's also tileserver-gl fork that migrated to use maplibre native, I think.It might even be a hosted project, IDK

Reply:
Oh, yes! I'm surprised TileServer GL wasn't in here already! Can't find a fork, but the original is a MapTiler project that's using MapLibre Native so I'll add that.